openstack/arc/templates/api-ingress.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
   {{- if .Values.api.ingress.vice_president }}
```

Comment:
Maybe change parameter name to make it less confusing.

Reply:
👍 @ArtieReus